pages/index.tsx

```diff
@@ -367,15 +370,27 @@ const InstallSection = () => {
   );
 };
 
+const RandomEntriesFromArray = (sourceArray: SimpleEntry[], count: number) => {
+  const source = sourceArray;
```

Comment:
Wanted to copy this array like `[...sourceArray]`, but it gave me some error with `Circular references cannot be expressed in JSON`, so there is some React or next mechanism that I clearly dont understand, but the selection seems to work fine without the copy, so I took it out.